server/impl/meetup.go

```diff
@@ -595,7 +590,7 @@ func (i *Implementation) PatchMeetupIDAttendee(params operations.PatchMeetupIDAt
 	id := session.Values[UserID]
 	attendeeID, err := db.UserIDFromString(params.PatchMeetupAttendeeBody.Attendee)
 	status := params.PatchMeetupAttendeeBody.AttendeeStatus
-	if attendeeID != 0 && err != nil {
+	if err != nil && params.PatchMeetupAttendeeBody.Attendee != "" {
```

Comment:
If someone put in "as;dlfkj" as an ID, this used to not trigger because the error condition [ParseUint](https://golang.org/pkg/strconv/#ParseInt)